### PR TITLE
Added raw data files and documentation on method for grouping

### PR DESCRIPTION
**Pull request by iglesias-cardinale**
# Added two folders

1) ## Data
### Added three raw data files:

- ```raw_data_table1.md``` is the data from table one in Hubble's paper.
- ```raw_data_table1_grouped.md``` is the data from table one in Hubble's paper sorted based on the first column.
- ```raw_data_table2.md``` is the data from table two in Hubble's paper. This one isn't particularly useful.
- 
### Added plotting code

- ```data_plotter.ipynb``` was used to plot the data to make sure the grouping was done correctly. It could also be built upon to add the curve fitting later.

2) ## Doumentation

- ```group_selection.md``` gives motivation and methods for grouping the data
- ```reproduced_figure1.png``` is a figure produced in ```data_plotter.ipynb```  that recreates the data display in figure 1 of Hubble's paper

### CODE REVIEW

**Comment by avgagliardo** [comment]
I'll request to review this code for tomorrow if no one else is in line for it.

**Comment by HeshamElsaman** [comment]
@avgagliardo great, I assigned it to you to review then.

File: Data/data_plotter.ipynb
Comment (avgagliardo) [inline Data/data_plotter.ipynb:1]: Reviewed, but I'll assume this is just a temporary workspace and not a file meant to be pushed

File: Data/raw_data_table1.md
Comment (avgagliardo) [inline Data/raw_data_table1.md:1]: Not rendering as a table

File: Data/raw_data_table1_grouped.md
Comment (avgagliardo) [inline Data/raw_data_table1_grouped.md:1]: Not rendering as a table

File: Data/raw_data_table2.md
Comment (avgagliardo) [inline Data/raw_data_table2.md:1]: Not rendering as a table

File: Documentation/group_selection.md
Comment (avgagliardo) [inline Documentation/group_selection.md:1]: looks good in terms of markdown rendering, but the link to https://github.com/ubsuny/CP1-24-HW4/blob/a04b0d2dc5e79a131f9ce6c6131e630f79c13ae6/workspaces/CP1-24-HW4/Data/raw_data_table1_grouped.md throws a 404 so either the file is not located at the location (sometimes the case for temporary files that may have moved), or perhaps the url is malformed.

File: Documentation/reproduced_figure1.png
Comment (avgagliardo) [inline Documentation/reproduced_figure1.png:1]: looks great

**Review by avgagliardo — CHANGES_REQUESTED** [review]
For now I will politely request the following changes:
- PR has several tables that are not rendering as proper markdown. 
- The following URL is nonfunctional: https://github.com/ubsuny/CP1-24-HW4/blob/a04b0d2dc5e79a131f9ce6c6131e630f79c13ae6/workspaces/CP1-24-HW4/Data/raw_data_table1_grouped.md

Otherwise the plot and doc look good, great work!

**Comment by avgagliardo** [comment]
@iglesias-cardinale let me know when you want me to review this again. I am doing my best to watch for it.

File: Data/data_plotter.ipynb
Comment (iglesias-cardinale) [reply Data/data_plotter.ipynb:1]: No, it doesn't need to be pushed. But I'm not sure how to remove it from the pull request.

File: Data/raw_data_table1.md
Comment (iglesias-cardinale) [reply Data/raw_data_table1.md:1]: The intention for the md files wasn't to render as tables. They were just presenting the data from table 1 of the paper, table 2 of the paper, and a grouped/organized version of table 1. I can see how the naming convention is misleading. They're actually specifically organized to be callable by np.loadtxt, so not sure if I should change these.

**Comment by avgagliardo** [comment]
Not sure if this is ready yet, but ill give it another review in about 30-60 mins (just in case)

File: Data/data_plotter.ipynb
Comment (avgagliardo) [reply Data/data_plotter.ipynb:1]: [Here](https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request) is an example for removing files, but its probably okay for now. 

If you want to delete it after the merge and do another PR, that works too. I'll just push this one for now so it doesn't hold things up.

File: Data/raw_data_table1.md
Comment (avgagliardo) [reply Data/raw_data_table1.md:1]: If its not supposed to be rendered as markdown, I would consider naming it as a .txt file for clarity but its okay now that I know its not supposed to be rendered as a table

**Review by avgagliardo — APPROVED** [review]
Changes look good, approving for merge

**Comment by HeshamElsaman** [comment]
Great, merging.

**Pull request merged**